src/framework/components/anim/component-layer.js

```diff
@@ -89,6 +93,23 @@ class AnimComponentLayer {
         }
     }
 
+    /**
+     * @function
+     * @name AnimComponentLayer#transition
+     * @description Transition to any state in the current layers graph. Transitions can be instant or take an optional blend time.
+     * @param {string} to - The state that this transition will transition to.
+     * @param {number} [time=0] - The duration of the transition in seconds.
+     * @param {number} [transitionOffset=null] - If provided, the destination state will begin playing its animation at this time. Given in normalised time, based on the states duration & must be between 0 and 1.
```

Comment:
I have a funny feeling that our JSDoc output ignores the default parameter values. Might wanna check that. Generally, the standard across our JSDoc comments is to say something like `Defaults to 0.`.

Reply:
> A question on these two new parameters on
> `pc.AnimComponent#assignAnimation(nodeName, animTrack, layerName, speed, loop);`
> 
> * just double checking - those parameters won't override values on a shared class between instances, but only do it for this specific anim component instance? Meaning if I have two characters using the same state graph, only the one I set it to will change?

Yep that's right.